heft-plugins/heft-storybook-plugin/src/StorybookPlugin.ts

```diff
@@ -117,14 +127,28 @@ export class StorybookPlugin implements IHeftPlugin<IStorybookPluginOptions> {
         '(EXPERIMENTAL) Used by the "@rushstack/heft-storybook-plugin" package to launch Storybook.'
     });
 
+    const storybookStaticBuildParameters: IHeftFlagParameter = heftSession.commandLine.registerFlagParameter({
+      associatedActionNames: ['build'],
+      parameterLongName: '--storybook',
+      description:
+        '(EXPERIMENTAL) Used by the "@rushstack/heft-storybook-plugin" package to start static build Storybook.'
+    });
+
     heftSession.hooks.build.tap(PLUGIN_NAME, (build: IBuildStageContext) => {
-      if (!storybookParameters.actionAssociated || !storybookParameters.value) {
+      if (
+        (!storybookParameters.actionAssociated || !storybookParameters.value) &&
+        (!storybookStaticBuildParameters.actionAssociated || !storybookStaticBuildParameters.value)
+      ) {
         this._logger.terminal.writeVerboseLine(
           'The command line does not include "--storybook", so bundling will proceed without Storybook'
         );
         return;
       }
 
+      this._modulePath = storybookStaticBuildParameters.actionAssociated
```

Comment:
If these options are made "optional", then we should throw if still undefined here, since the modulePath would be needed to run.

Reply:
Good catch, updated